docs/api-docs-openapi.yaml

```
@@ -0,0 +1,3820 @@
+        transaction hash if filled.
+         
+        This endpoint loads data queried by an indexing service that polls
+        relevant events on a 10-second cadence. 
+
+        Users should therefore expect an average latency of 1 to 15 seconds
+        after submitting a deposit to see the status changed in this endpoint. 
+
+        This delay comes from the time it takes for the internal indexing to
+        include the deposit transaction.
+      parameters:
+        - name: originChainId
+          in: query
+          required: false
+          description: >
+            Chain Id where the deposit originated from. This must be used in
+            conjunction with depositId. depositTxHash is not required when using
+            this parameter.
+          schema:
+            allOf:
+              - $ref: '#/components/schemas/ChainId'
+          examples:
+            default:
+              value: 137
+          x-gitbook-description-html: >-
+            <p>Chain Id where the deposit originated from. This must be used in
+            conjunction with depositId. depositTxHash is not required when using
+            this parameter.</p>
+        - name: depositId
+          in: query
+          required: false
+          description: >
+            The deposit id that is emitted from the DepositV3 function call as a
+            V3FundsDeposited event. This must be used in conjunction with
+            originChainId. depositTxHash is not required when using this
+            parameter.
+          schema:
+            type: integer
+            minimum: 0
+          examples:
+            default:
+              value: 1349975
+          x-gitbook-description-html: >-
+            <p>The deposit id that is emitted from the DepositV3 function call
+            as a V3FundsDeposited event. This must be used in conjunction with
+            originChainId. depositTxHash is not required when using this
+            parameter.</p>
+        - name: depositTxHash
+          in: query
+          required: false
+          description: >
+            The deposit transaction hash that is emitted from the DepositV3
+            function call as a V3FundsDeposited event. If you are using this,
+            you donot need the above parameters. 
+          schema:
+            type: string
+          examples:
+            default:
+              value:                '0x56a86610aad2a653161c9dd4b4972f948198fd02ae9589cfac1d57bd86a156c1'
+          x-gitbook-description-html: >-
+            <p>The deposit transaction hash that is emitted from the DepositV3
+            function call as a V3FundsDeposited event. If you are using this,
+            you donot need the above parameters.</p>
+      responses:
+        '200':
+          description: Lifecycle of a transaction
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  fillStatus:
```

Should this match the field name of the response?
```suggestion
                  status:
```